IQKeyboardManager/IQKeyboardManager.m

```diff
@@ -420,11 +420,6 @@ -(void)setEnableAutoToolbar:(BOOL)enableAutoToolbar
     {
         [self addToolbarIfRequired];
     }
-    //Else removing toolbar.
-    else
-    {
-        [self removeToolbarIfRequired];
```

Comment:
Well, I removed it because it would have no effect anymore, because if `[self privateIsEnableAutoToolbar]` is `NO`, `[self removeToolbarIfRequired]` would return immediately ;)